pkg/storer/internal/reserve/reserve.go

```diff
@@ -45,6 +45,7 @@ type Reserve struct {
 func New(
 	baseAddr swarm.Address,
 	st transaction.Storage,
+	maxCapacityHeight int,
```

Comment:
internal reserve does not need to know about maxCapacityHeight

Reply:
fixed